scripts/verify-brackets.js

```diff
@@ -0,0 +1,11 @@
+const config = require("../server/routes/config");
+const isReachable = require("is-reachable");
+var brambleServer = config.editorHOST;
```

Comment:
nit: this should be a `const` since we don't change its value at any point.

Reply:
Oh ok makes sense, Thanks!